pkgs/top-level/python-packages.nix

```diff
@@ -10169,6 +10169,8 @@ in {
 
   yamale = callPackage ../development/python-modules/yamale { };
 
+  yamlfix = callPackage ../development/python-modules/yamlfix { };
```

Comment:
If this is only a cli and not really used as a python library we can move it completely to a cli tool.

Reply:
It can be used as both:

As a command line tool:

```bash
$: yamlfix file.yaml
```

As a library:

```python
from yamlfix import fix_files

fix_files(['file.py'])
```

Does that still imply that a change is needed?

Reply:
then it is fine to be here.